common/likeable-model.js

```diff
@@ -58,7 +57,7 @@ export const LikeableModel = Base => class extends Base { //eslint-disable-line
      */
     likeCount() {
         // This creates backwards compatibility for when we stored userId's in an array on the liked object
-        return _.isArray(this._likeCount) ? this._likeCount.length : this._likeCount || 0;
+        return Array.isArray(this._likeCount) ? this._likeCount.length : this._likeCount || 0;
```

Comment:
this can actually just return `this._likeCount`

Reply:
Done!

Reply:
what do you think about removing the _ and just removing this method all together? It may be best to do this now before usage goes up and while we are still in a pre v1 state.

Reply:
Same thing with the comments count. Since we are now returning only a param in the document it seems a bit redundant, but some people might be a bit squemish about getting something from a param that starts with underscore not to mention that there are lint rules that complain about that.
But with all that said, I think it will be better for performance if people who need it get it directly from the document instead of through the method.